src/styles/main.scss

```diff
@@ -1,3 +1,25 @@
+* {
+  box-sizing: border-box;
+}
+
+p,
+ul,
+li,
+h2,
+a,
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+main {
+  margin: 0 auto;
 }
```

Comment:
create `reset.css` and move these styles there

Reply:
https://valeriiafomenko.github.io/layout_catalog/
https://valeriiafomenko.github.io/layout_catalog/report/html_report/